Cargo.toml

```diff
@@ -15,9 +15,8 @@ categories = ["command-line-utilities"]
 thiserror = "1.0.30"
 log = "0.4.16"
 env_logger = "0.9.0"
-# TODO: Update `parquet` and `arrow` to `40.0.0` when they are released
```

Comment:
thank you! could you also bump the `version` to `0.3.1`?

Reply:
Sure!